RocketChat/Rocket.Chat
# fix: Capitalized letters are bypassing the alphabetical ordering on Admin lists

## Proposed changes (including videos or screenshots)


The alphabetical ordering on these lists is listing capitalized words “first”, and only then listing non-capitalized ones.



## Issue(s)


## Steps to test or reproduce


Create a few rooms/users that should follow alphabetical ordering in a clear way for ease of testing.

1. E.g.: Carry, Chelsea, Cicero, Colleen, and Columbus.
2. Choose one of the rooms/users and change its name/username to a non-capitalized word.
3. For this example, I’ll change “Cicero” to “cicero”
4. Go to the User listing on Admin Setting 

### Expected Behavior

The Users created in the Steps above should appear in this order: 

- Carry; Chelsea; cicero; Colleen; Columbus;

The alphabetical order should be kept despite the capitalized letters.

### Current Behavior

The capitalized words are listed first, The same names as the above section would list:

- Carry; Chelsea; Colleen; Columbus; cicero;

## Further comments

[TC-525]


[TC-525]: https://rocketchat.atlassian.net/browse/TC-525?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
